packages/angular/src/index.ts

```diff
@@ -0,0 +1,68 @@
+import { DOCUMENT } from '@angular/common';
+import {
+	DestroyRef,
+	Directive,
+	ElementRef,
+	inject,
+	Input,
+	Output,
+	EventEmitter,
+} from '@angular/core';
+import { Draggable, type DragEventData, type DragOptions } from '@neodrag/vanilla';
+
+@Directive({ selector: '[neoDraggable]', standalone: true })
+export class NeoDraggable {
+	private _document = inject(DOCUMENT);
+	private _elRef = inject<ElementRef<HTMLElement>>(ElementRef);
+
+	private _draggable: Draggable | null = null;
+
+	// NOTE: '' type is to allow for using neoDraggable as-is without passing in any options
+	@Input() set neoDraggable(options: DragOptions | '') {
```

Comment:
Couldn't this be `options: DragOptions = {}`?

Reply:
Unfortunately no for 2 reasons:
- Setter cannot have default argument
- Angular Language Service perceives the type of the value passed in to an Input as `''` if used like an HTML Attribute `<div neoDraggable></div>`

Reply:
Aha!!